### PR TITLE
fix #2925- Loader animation added 

### LINKED ISSUE #2925: New loader animation
## Description
Loader animation for community-app when it starts.

## Steps to Reproduce
Please add as much detail as possible to reproduce the error/issue.
Don't assume the reader knows anything about the Mifos X Web App.

**Old design:** 
screenshot from 2018-03-06 10-13-50

**New design**

https://jsfiddle.net/dumptyd/32hregpo/

### PR DESCRIPTION
**Pull request by muddlebee**
## Description
New loader animation. Please refer issue for more details

also changes due to #2946 there are some minor changes in some lines for
`styles.css` which resulted in a large diff.

## Related issues and discussion
#2925 
## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Review by botraunak — CHANGES_REQUESTED** [review]
Why do we need to update the gruntfile for changing the loader animation?

**Comment by botraunak** [comment]
@anweshknayak  I have merged the grunt file changes please rebase and submit new PR without the gruntfile changes.

**Comment by muddlebee** [comment]
@botraunak I think you can directly have `rebase and merge` option in the github itself. Can you do it?

**Comment by muddlebee** [comment]
@botraunak done.
Grunt file changes to include `compass:dev` for everytime we make changes in scss file. It will automatically convert to css. I forgot to include this in previous PR where grunt file changes were made

**Comment by botraunak** [comment]
@anweshknayak The diff is not visible when performing the rebase and merge, Can you do this and update the PR, and all the other PRs which rely on it. It's necessary to have only the diff the relates to the issue and not related issues.

**Comment by muddlebee** [comment]
@botraunak diff it is due to the changes done in Grunt file configurations related to `compass:dev` . It lints the css file and add comments from which scss file it was added. Diff is not visible because the change is large. Click it the changes will appear.

And this is not due to any other PRs or issues

**Comment by muddlebee** [comment]
Also if diff is not appearing. Could you pull this PR locally and check ?

**Review by botraunak — APPROVED** [review]
Looks okay to me

File: Gruntfile.js [hunk 1]
```diff
@@ -418,7 +418,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-gh-pages')
 
   // Run development server using grunt serve
-  grunt.registerTask('serve', ['clean:server', 'copy:server', 'connect:livereload', 'watch']);
+  grunt.registerTask('serve', ['clean:server', 'copy:server', 'compass:dev', 'connect:livereload', 'watch']);
```
Comment (botraunak) [inline Gruntfile.js:421]: Isn't this already included in the watch task check under css, of the watch task definition!

**Review by botraunak — CHANGES_REQUESTED** [review]

File: Gruntfile.js (hunk 1 above)
Comment (botraunak) [reply Gruntfile.js:421]: Now that you have fixed the compass:dev I don't think you need to call this in the serve task explicitly as it's already being done in the watch task :)

**Comment by muddlebee** [comment]
@botraunak done

**Comment by muddlebee** [comment]
@botraunak I tried removing `compass:dev` from grunt serve task. The difference is we have to keep running grunt serve for changes in scss-> css to incorporate. Otherwise if we make changes and run `grunt serve` the changes don't appear.

If we already include `compass:dev` in serve task and make changes and scss and start `grunt serve` changes are being incorporated in scss otherwise it is not. Hope it makes sense.

**Review by botraunak — APPROVED** [review]

**Pull request merged**